File: CodeChef/Beginner/11SquareRoot/sqroot.java
import java.util.Scanner;
import java.lang.Math;

class sqroot {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, x;
        n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            System.out.println((int) Math.floor(Math.sqrt(x)));

        }
        sc.close();
    }
}

File: CodeChef/Beginner/11SquareRoot/sqroot.cpp
#include <cmath>
#include <iostream>
using namespace std;
class sqroot {
public:
};

int main() {
  int n, x;
  cin >> n;

  for (int i = 0; i < n; i++) {
    cin >> x;

    cout << floor(sqrt(x)) << endl;
  }
}
